### ISSUE TITLE
[🐛] Broken lint:android task

### ISSUE THREAD

**Issue by dylancom** [invertase/react-native-google-ads#22]
`Unrecognized option: --add-exports
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
Error: google-java-format exited with exit code 1.`

Seems related to: https://github.com/invertase/nodejs-google-java-format/issues/2
Although there has been reported that it happens occasionally, to me it happens all the time.

**Comment by mikehardy** [comment]
That's odd, definitely worked for me last night although had one flaky run locally.

I need more environment details if you have them
Also, how did you get that error information? I did not dig too deeply when initially hacking together the android lint check but could not determine the root cause and this appears it might be it. Must have missed something obvious in error reporting in the underlying code I snuck away with from clang-format

**Comment by dylancom** [comment]
I get this error info after running `yarn run lint:android` in my terminal.

**Comment by mikehardy** [comment]
Found it. Requires JDK11, I've already moved everywhere including github workflows so was a latent error in expectations.
I can put guardrails around that, but that's the root cause for you locally

JDK11 is becoming a hard requirement because android compileSdkVersion (not target - compile!) requires it or you get other wierd bugs (one where buildtools 31 is "corrupt", associated with gradle 7 / android gradle plugin 7) and one where there's an internal compiler bug about a module symbol. Time to update - `brew tap AdoptOpenJdk/openjdk && brew install adoptopenjdk11` :-)
